src/Test/WebTestCase.php

```diff
@@ -309,13 +309,14 @@ protected function isBackupUpToDate(array $classNames, string $backup): bool
      * class path.
      *
      * @param array  $classNames   List of fully qualified class names of fixtures to load
+     * @param bool   $append
      * @param string $omName       The name of object manager to use
      * @param string $registryName The service id of manager registry to use
      * @param int    $purgeMode    Sets the ORM purge mode
      *
      * @return null|AbstractExecutor
      */
-    protected function loadFixtures(array $classNames = [], ?string $omName = null, string $registryName = 'doctrine', ?int $purgeMode = null): ?AbstractExecutor
+    protected function loadFixtures(array $classNames = [], bool $append = false, ?string $omName = null, string $registryName = 'doctrine', ?int $purgeMode = null): ?AbstractExecutor
```

Comment:
generally this method has too many parameters already from my point of view. it would be good to think about how to refactor it to have less parameters.

Reply:
Since we are targeting 2.x and BC are allowed, it's a good moment to do that, yes! :+1:

Reply:
But, has it to be in this PR or another one?